.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+# Pull Request Template
```

Comment:
@Mastersam07 can this be changed to `Hey, I just made a Pull Request` or maybe something else but this :sweat_smile:

Reply:
Okay 😄